uint32 PingerFrequency
uint32 LowestFrequency
uint32 HighestFrequency
float32 FrontThreshold
float32 PowerThreshold
uint32 DelayObserve
float32 AmplifierNormalizedGain
---
bool result
